### ISSUE TITLE
SyncTeX for Evince

### ISSUE THREAD

**Issue by edwinksl** [thomasjo/atom-latex#162]
I am trying to make SyncTeX work for the Evince PDF viewer. I have set the `Custom PDF Viewer Path` to `/usr/bin/evince`  and was hoping that SyncTeX would work out of the box but this isn't the case. Any tips for making SyncTex play well with Evince?

**Comment by peterbabic** [comment]
Still good for you, I did set the same thing, but atom opens the resulting PDF in source textual form. I do not understand why. (using XFCE linux environment, Okular has too much KDE dependencies, so I do not want to use it).

**Comment by yitzchak** [comment]
In order to enable SyncTeX with Evince we will probably need to have an opener class specifically for Evince. Currently, [SyncTeX support in Evince](https://help.gnome.org/users/evince/stable/index.html.en#synctex) works with gedit and can work with vim by using [evince_dbus.py](https://git.gnome.org/browse/gedit-plugins/tree/plugins/synctex/synctex/evince_dbus.py) which is supplied as a sample by gedit.

What do you think @thomasjo? Evince is the default viewer in Gnome 3 and hence my usual viewer, so I can certainly put it on my long term todo list, but supporting it will most likely result in a dbus dependency.

**Comment by thomasjo** [comment]
I definitely see the benefit of adding this. My only concern is adding in a lot of additional complexity just to support a PDF viewer for a subset of users. If we can keep it fairly straight-forward, yet robust, then we should add support for Evince.

**Comment by yitzchak** [comment]
Resolve #162

**Comment by yitzchak** [comment]
This feature has been added and should be in the next release.

**Comment by cbrnr** [comment]
How do I trigger forward and backward search? I can't seem to figure it out, neither Ctrl+click in Evince nor Ctrl+Alt+S in Atom is working...

**Comment by yitzchak** [comment]
@cbrnr xdg-open is the fallback opener which will use Evince but not support SyncTeX. Trying putting the path to Evince in the settings. If that does not work, please open a new issue since this issue was to add support for Evince.

**Comment by cbrnr** [comment]
@yitzchak according to the [Wiki](https://github.com/thomasjo/atom-latex/wiki/Supported-Viewers), Evince should support both forward and backward search. Evince is opening fine with the default settings, but I can't seem to figure out how fw/bw search works...

**Comment by yitzchak** [comment]
@cbrnr You correct, Evince supports both forward and backward search. In order for it work `xdg-open` cannot be used. The setting 'Evince Path` activates Evince. You also need SyncTeX turned on.

**Comment by cbrnr** [comment]
Yes, I have all of this set up correctly, but searching doesn't work. Are there any settings I need to change in Evince?

**Comment by yitzchak** [comment]
What are the other settings in `atom-latex` set to? Output directory, etc.

**Comment by cbrnr** [comment]
I think I left all settings at their defaults. Shell escape is enabled, and so is SyncTeX. The output directory is blank, move result to source directory is enabled. Evince path is blank, i.e. set to the default.

Evince is opening automatically after I build the document, but I don't know how to sync. I tried right-clicking in Atom and selecting "LaTeX Sync", but this doesn't do anything. I tried ctrl-clicking in Evince, and this doesn't do anything either. I even activated the gedit SyncTeX plugin, but I don't think this is necessary (or is it?).
